### ISSUE TITLE
Could plotly graph support change y axis automatically?

### ISSUE THREAD

**Issue by foolcage** [plotly/plotly.py#1741]
If I want to show diffent range y trace in the same graph and make them comparable,I could use difffent y axis for them. my question is:could this be supported in the graph tools automatically?just like zoom in/out or pan functions?

**Comment by Mark531** [comment]
Same request here... It's a highly anticipated feature of plotly but I don't have the feeling that it's considered for development. At least, we're interested in workarounds.

**Comment by nicolaskruchten** [comment]
I'm not sure I understand the question here, can you rephrase please? Are you asking about multiple y-axes? There's an example with many Y-axes here: https://plot.ly/python/multiple-axes/#multiple-axes

**Comment by Mark531** [comment]
I admit that the question wasn't so clear, but the issue that has been raised by many people is that zooming with the x-range slider doesn't rescale the y-axis. But I found a very easy workaround yesterday: it's enough to remove the x-range slider! This way, zooming on any part of the graph rescales the y-axis and the visualization stays at the right scale.

**Comment by gvwilson** [comment]
Hi - we are currently trying to tidy up Plotly's public repositories to help us focus our efforts on things that will help users most. Since this issue has been sitting for several years, I'm going to close it. If it's still a concern, we'd be grateful if you could open a new issue (with a short reproducible example if appropriate) so that we can add it to our backlog. Thanks for your help - @gvwilson
